skeleton/gitops-template/components/http/base/rhoai/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+kind: ServiceAccount
```

Comment:
also missed this on the other PR, do we need this SA?

Reply:
Yes, it's needed so that the RHOAI notebook doesn't use the default sa.